rmi/servercode/ResImpl/ResourceManagerImpl.java

```diff
@@ -434,27 +435,6 @@ public boolean deleteCustomer(int id, int customerID)
         } // if
     }
 
-

```

Comment:
We had this method commented out for the last two projects, figured we could just scrap it